### PR TITLE
Add hospital admissions vaccine incidence schema

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard17**

### CODE REVIEW

File: packages/app/schema/nl/__index.json [hunk 1]
```diff
@@ -20,11 +20,13 @@
     "hospital_nice",
     "hospital_nice_per_age_group",
     "hospital_vaccination_status",
+    "hospital_vaccine_incidence_per_age_group",
     "infectious_people",
     "intensive_care_lcps",
     "intensive_care_nice",
     "intensive_care_nice_per_age_group",
     "intensive_care_vaccination_status",
+    "intensive_care_vaccine_incidence_per_age_group",
```
Comment (VWSCoronaDashboard16) [inline packages/app/schema/nl/__index.json:29]: I think this does not belong in this PR

**Review by VWSCoronaDashboard1 — APPROVED** [review]
